scripts/mkcloud

```diff
@@ -197,11 +197,11 @@ function error_exit
             for node in $(crowbar machines list | grep ^d) ; do
             (
                 echo "Collecting supportconfig from $node"
-                timeout 400 ssh $node supportconfig | wc
+                timeout 400 ssh $node supportconfig -l -I 0 | wc
```

Comment:
1. code duplication => maybe move those params or the whole call into a variable.
Then we can even allow users to override the defaults via env

2. disabling the limit might collect unexpected amount of logs (in case of runaway writers), I'd prefer a high number instead. e.g. 5000